test/Azure.Functions.Cli.Tests/E2E/StartTests.cs

```diff
@@ -389,6 +391,76 @@ await CliTester.Run(new RunConfiguration
             }, _output);
         }
 
+        [Fact]
+        [Trait(TestingTraits.TraitName.Category, TestingTraits.TestCategory.RequiresNestedInProcArtifacts)]
```

Comment:
With the suggested change mentioned in the other PR, it would look like this:

`[Trait(TestTraits.Category, TestTraits.RequiresNestedInProcArtifacts)]`

Reply:
Actually, one more suggestion on this: can you switch to using groups for this? I want to have these all in the E2E category @aishwaryabh

Reply:
Switched to using groups!